src/assets/css/themes/darkly.css

```diff
@@ -1 +1,2 @@
+@import "hljs/dark.css";
```

Comment:
How about instead of editing every css file, load the appropriate light / dark one based on prefers-color-scheme? 

You could do that in this file somewhere: https://github.com/LemmyNet/lemmy-ui/blob/main/src/shared/components/app/theme.tsx#L39

Reply:
There are a bunch of hljs themes, so it seems like a shame to only include two =\

I think the ultimate solution would be to have a user-setting like for the site-themes.

I could implement it using local storage while we debate if it warrants inclusion in the API proper